mobile.js

```diff
@@ -68,11 +67,6 @@ function showError() {
 }
 
 form.addEventListener('submit', (e) => {
```

Comment:
- [ ] **[OPTIONAL]** In case you noticed, I am able to submit the form with empty input **name** and **message** fields. For UX purpose and to avoid receiving unnecessary data from the client-side, it would be nice if you validate the input of other fields, generally speaking.

![msg-name-valid](https://user-images.githubusercontent.com/36542986/136426864-03f9a0c7-5810-4637-bbb9-4fd2a71821b1.jpg)

This happens probably because you added a **`novalidate`** attribute to your form at some point during previous projects. It would be worth it if you fix this form behaviour.

![novalidate](https://user-images.githubusercontent.com/36542986/136426637-8dae5f84-c900-46b4-8e34-a50e61c2b80e.png)